mailpoet/lib/EmailEditor/Integrations/MailPoet/EmailEditor.php

```diff
@@ -56,4 +57,27 @@ public function extendEmailPostApi() {
       'schema' => $this->emailApiController->getEmailDataSchema(),
     ]);
   }
+
+  public static function getEmailDefaultContent(): string {
```

Comment:
Was not sure about this. It needs to be a function because of localisation, but perhaps there is a better place for this method to live?

It needs access to `Env::$assetsUrl` for images.

Reply:
I think it is good 👍 . I see this starter content as a MailPoet-specific feature, so Integration/MailPoet namespace makes perfect sense. 
This is the kind of static stuff that could be a constant, but we need the localization so the static method is fine.

Reply:
I think we should not use `Env:$assetsUrl` here but use the same approach as it is in [the form templates](https://github.com/mailpoet/mailpoet/blob/926620e8f82ca691f1c11f6e973a2f86a6117522/mailpoet/lib/Form/Templates/FormTemplate.php#L147-L149). Those images are stored in [the SVN plugin repository](https://github.com/mailpoet/mailpoet/tree/926620e8f82ca691f1c11f6e973a2f86a6117522/mailpoet/plugin_repository/assets) and are not a part of the plugin package. So, the plugin build can be a bit smaller.
What do you think @costasovo and @mikejolley?

Reply:
I tried to add it to the acceptance tests and it seems that we might only need to add `wp config set MAILPOET_USE_CDN false --raw` into [this place](https://github.com/mailpoet/mailpoet/blob/eea5ca0b832e14db56b63ea789a2ab5c5f4729c9/mailpoet/tests/docker/codeception/docker-entrypoint.sh#L161-L165). So I guess we could add it. I see it as an improvement to the tests